pkg/controller/clusterinstall/controller.go

```diff
@@ -405,16 +405,13 @@ func (s *jobSyncStrategy) DoesOwnerNeedProcessing(owner metav1.Object) bool {
 		return false
 	}
 
-	// cannot run ansible jobs until the ClusterVersion has been resolved
-	if cluster.ClusterDeploymentStatus.ClusterVersionRef == nil {
-		// staebler: Temporary work-around until we have a controller that sets
-		// the ClusterVersionRef for cluster-api clusters.
-		_, err := s.controller.clustopClient.ClusteroperatorV1alpha1().
-			ClusterVersions(cluster.ClusterDeploymentSpec.ClusterVersionRef.Namespace).
-			Get(cluster.ClusterDeploymentSpec.ClusterVersionRef.Name, metav1.GetOptions{})
-		if err != nil {
-			return false
-		}
+	// TODO: staebler: Temporary work-around until we have a controller that sets
```

Comment:
I wonder if this is needed here given that we are referencing the ClusterVersion from the ClusterDeployment and we check for its existence in the ClusterDeployment controller before attempting to create the Cluster resource.

Reply:
Agreed, I think the copy to the cluster provider spec is the new "resolved".